.github/workflows/flutter.yml

```
@@ -1,4 +1,7 @@
 name: "Build for all platforms"
+permissions: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)